conference.js

```diff
@@ -1394,10 +1401,15 @@ export default {
         room.on(ConferenceEvents.ENDPOINT_MESSAGE_RECEIVED,
             (participant, payload) => {
                 switch(payload.type) {
-                    case this.commands.defaults.CONNECTION_QUALITY:
-                        ConnectionQuality.updateRemoteStats(participant.getId(),
-                            payload.values);
+                    case this.commands.defaults.CONNECTION_QUALITY: {
+                        let id = participant.getId();
+                        ConnectionQuality.updateRemoteStats(
+                            id,
+                            payload.values,
+                            APP.UI.getRemoteVideoType(id),
+                            APP.UI.isRemoteVideoMuted(id));
```

Comment:
Because we do not have access to the remote track here or inside ConnectionQuality. Or I'm missing something?